cmd/qat_plugin/dpdkdrv/dpdkdrv.go

```diff
@@ -249,6 +253,57 @@ func (dp *DevicePlugin) getDpdkMounts(dpdkDeviceName string) []pluginapi.Mount {
 	}
 }
 
+func getDeviceCapabilities(device string) string {
+	devID, err := getDeviceID(device)
+	if err != nil {
+		return defaultCapabilities
+	}
+
+	devicesWithCapabilities := map[string]struct{}{
+		"4941": {},
```

Comment:
Maybe a comment to explain what "4941" is?

Reply:
I'll add a comment.

Reply:
added